src/features/canvass/l10n/messageIds.ts

```diff
@@ -153,9 +153,13 @@ export default makeMessages('feat.canvass', {
         title: m('All time'),
       },
       header: {
+        householdVisits: m('Household Visits'),
         households: m('Households'),
         locations: m('Locations'),
+        successfulHouseholdVisits: m('Successful Household Visits'),
```

Comment:
Same is true here as well, but I suggest keeping the label a bit brief to make sure it fits:

```suggestion
        householdsSuccessful: m('Successfully visited'),
```